docs/services/auth.rst

```diff
@@ -65,6 +65,18 @@ Auth Responses
    :members:
    :show-inheritance:
 
+.. autoclass:: AuthorizationCodeTokenResponse
+   :members:
+   :show-inheritance:
+
+.. autoclass:: RefreshTokenResponse
+   :members:
+   :show-inheritance:
+
+.. autoclass:: ClientCredentialsTokenResponse
```

Comment:
Should these be alphabetical?

Reply:
Arguably? Either alphabetical or in some other reasonable sorted order.
I was trying to put them in order from "most used to least used", but I actually think that necessitates putting the OAuthDependentTokenResponse below these. Would you be in favor of such a change?

Reply:
Sorting by assumed usage stats is fine, just wanted to understand if there was an ordering decision made here.